### PR TITLE
Uniform styling of settings page

### PR DESCRIPTION
**Pull request by frederike-ramin**

### CODE REVIEW

File: src/SettingsView/EntityMappingListItem.js [hunk 1]
```diff
@@ -56,43 +78,26 @@ class EntityMappingListItem extends ConnectionComponent {
...
-							);
-						})}
-					</Col>
-				</Row>
-			</ListItem>
+			<Table selectable={false}>
+				<TableHeader displaySelectAll={false} adjustForCheckbox={false}>
+					<TableRow>
+						<TableHeaderColumn className="mapping-headline">{this.props.eventType.Name}</TableHeaderColumn>
+						<TableHeaderColumn className="mapping-headline">{this.getEntityTypeName(this.props.mapping.EntityTypeId)}</TableHeaderColumn>
```
Comment (julianweise) [inline src/SettingsView/EntityMappingListItem.js]: Line too long

File: src/SettingsView/EntityMappingListItem.js [hunk 2]
```diff
@@ -56,43 +78,26 @@ class EntityMappingListItem extends ConnectionComponent {
...
+						<TableHeaderColumn className="smallWidth">{this.getMappingActionButtons()}</TableHeaderColumn>
+					</TableRow>
+				</TableHeader>
+				<TableBody displayRowCheckbox={false}>
+					{this.props.mapping.EventEntityMappingConditions.map((condition, key) => {
+						return (
+							<TableRow key={key}>
+								<TableRowColumn className="mapping-condition" key={key + "EventTypeName"}>{this.getEventTypeAttributeName(condition.EventTypeAttributeId)}</TableRowColumn>
```
Comment (julianweise) [inline src/SettingsView/EntityMappingListItem.js]: Line too long

File: src/SettingsView/EntityMappingListItem.js [hunk 3]
```diff
@@ -56,43 +78,26 @@ class EntityMappingListItem extends ConnectionComponent {
...
+								<TableRowColumn className="mapping-condition" key={key + "EntityName"}>{this.getEntityTypeAttributeName(condition.EntityTypeAttributeId)}</TableRowColumn>
```
Comment (julianweise) [inline src/SettingsView/EntityMappingListItem.js]: Line too long

File: src/SettingsView/EventQueryListItem.js [hunk 4]
```diff
@@ -31,7 +31,8 @@ class EventQueryListItem extends Component {
 				</ConfirmationMessage>
 				<IconButton
 					tooltip="Edit this query"
-					href={Utils.getLink(`/settings/eventType/${this.props.eventType.Id}/eventQuery/${this.props.query.Id}/edit`)} >
+					href={Utils.getLink(`/settings/eventType/${this.props.eventType.Id}/eventQuery/${this.props.query.Id}/edit`)}
```
Comment (julianweise) [inline src/SettingsView/EventQueryListItem.js:34]: Line too long

File: src/SettingsView/EventTypeCard.js [hunk 5]
```diff
@@ -60,43 +63,91 @@ class EventType extends ConnectionComponent {
...
+							return (
+								<div>
+									<EntityMappingListItem
+										key={mapping.Id}
+										mapping={mapping}
+										deleteMapping={this.props.deleteMapping}
+										eventType={this.props.eventType}
+										eventTypeAttributes={attributes}/>
+									{index !== (entityMappings.length - 1) &&
+									<Divider key={index} style={{backgroundColor: config.colors.accent, marginTop: '20px', marginBottom: '20px'}} />}
```
Comment (julianweise) [inline src/SettingsView/EventTypeCard.js]: Line too long

**Review by julianweise — CHANGES_REQUESTED** [review]

File: src/SettingsView/EventQueryListItem.js (hunk 4 above)
Comment (frederike-ramin) [reply src/SettingsView/EventQueryListItem.js:34]: Nope.

**Review by julianweise — APPROVED** [review]

**Pull request merged**